internal/livechat/sdk/client.go

```diff
@@ -99,29 +99,36 @@ type registeredWebhook struct {
 
 type listWebhooksResponse []registeredWebhook
 
-func (c *LivechatSDKClient) MakeRequest(path string, method string, body []byte) ([]byte, error) {
+type transferChatRequest struct {
+	ID string `json:"id"`
+}
+
+func (c *LivechatSDKClient) MakeRequest(path string, method string, body []byte, authToken ...string) ([]byte, error) {
```

Comment:
This feels hacky for me. You accept token for some methods, for some others you don't. I uderstand the purpose, but I'm not a fan. Either accept token everywhere or create a separate sdk object for bot.